paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/activity/LaunchPaymentSheetCompleteActivity.kt

```diff
@@ -1,41 +1,212 @@
+                                    merchantDisplayName = merchantName,
+                                    customer = customerConfig,
+                                    googlePay = googlePayConfig,
+                                )
+                            )
+                        }
+                    }
                 )
             }
         }
     }
 }
+
+@Composable
+fun Receipt(
+    buyButtonEnabled: Boolean,
+    buyButtonClickListener: () -> Unit,
+) {
+    Surface(color = BACKGROUND_COLOR) {
+        Column(Modifier.fillMaxSize().padding(16.dp)) {
+            Text(
+                stringResource(R.string.cart_title),
+                style = MaterialTheme.typography.h5,
+                fontWeight = FontWeight.Bold,
+                modifier = Modifier.padding(bottom = 12.dp)
+            )
+            Card(Modifier.fillMaxWidth(1f)) {
+                Column(
+                    Modifier.fillMaxWidth(1f)
+                ) {
+                    ProductRow(HOT_DOG_EMOJI, R.string.hot_dog, "$0.99")
+                    ProductRow(SALAD_EMOJI, R.string.salad, "$8.00")
+                }
+            }
+            Column(
+                Modifier.fillMaxWidth(1f).padding(vertical = PADDING)
+            ) {
+                ReceiptRow(stringResource(R.string.subtotal), "$8.99")
+                ReceiptRow(stringResource(R.string.sales_tax), "$0.74")
+                TotalLine(Modifier.align(Alignment.CenterHorizontally))
+                ReceiptRow(stringResource(R.string.total), "$9.73")
+                BuyButton(buyButtonEnabled, buyButtonClickListener)
+            }
+        }
+    }
+}
+
+@Composable
+fun ProductRow(
+    productEmoji: String,
+    productResId: Int,
+    priceString: String,
+) {
+    Row(
+        Modifier.padding(PADDING),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+        Text(
+            productEmoji,
+            modifier = Modifier.padding(end = 4.dp),
+            fontSize = EMOJI_FONT_SIZE
+        )
+        Column(
+            Modifier.fillMaxWidth(ROW_LEFT)
+        ) {
+            Text(
+                text = stringResource(productResId),
+                fontSize = MAIN_FONT_SIZE
+            )
+            Text(
+                text = stringResource(R.string.product_subtitle, stringResource(productResId)),
+                fontSize = SUB_FONT_SIZE
+            )
+        }
+        Text(
+            text = priceString,
+            Modifier.fillMaxWidth(),
+            style = TextStyle.Default.copy(textAlign = TextAlign.Right),
+            fontSize = MAIN_FONT_SIZE
+        )
+    }
+}
+
+
+@Composable
+fun ReceiptRow(
+    description: String,
+    priceString: String,
+    color: Color = Color.Unspecified
+) {
+    Row(
+        Modifier.padding(vertical = PADDING),
+        verticalAlignment = Alignment.CenterVertically
+    ) {
+
+        Text(
+            text = description,
+            modifier = Modifier.fillMaxWidth(ROW_LEFT),
```

Comment:
is there a `ROW_START`?

Reply:
just renamed the variable to get us there